I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 24, in script
    sound "clock.mp3"
Exception: Sayer 'sound' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 24, in script
    sound "clock.mp3"
  File "C:\Users\Dima\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 890, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Dima\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 783, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'sound' is not defined.

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
FireStory 1.0
Sun Nov 26 12:26:16 2023
